core/test/api/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,259 @@
+// +build !ignore_autogenerated
```

Comment:
Why we need all these types duplication here?

Reply:
removed it